specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json

```diff
@@ -1473,6 +1473,16 @@
           "type": "string",
           "description": "The name of the managed resource group created by workspace RP in customer subscription if the workspace is CMK workspace",
           "readOnly": true
+        },
+        "privateLinkCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Count of private connections in the workspace",
+          "readOnly": true
+        },
+        "imageBuildCompute": {
```

Comment:
Adding a non-readonly property to an existing API version is regarded as a breaking change. What's the motivation for adding this?

Reply:
This is the new property which is already supported in our service.